app/assets/stylesheets/admin/icons.css.scss

```diff
@@ -1,3 +1,4 @@
 @import 'plugins/font-awesome';
 
 .icon-refund:before { @extend .icon-ok:before }
```

Comment:
what about `.icon-refund:before, .icon-credit:before { ... }` ?

Reply:
Higher maintenance cost? If I want to change credit to a different icon I have to split them out again. :trollface:

Reply:
Happy to keep as is? I will change it if you insist.

Reply:
no need to worry.